packages/web-components/fast-foundation/src/combobox/combobox.ts

```diff
@@ -206,9 +206,12 @@ export class Combobox extends FormAssociatedCombobox {
      * @public
      */
     @observable
-    public position: SelectPosition = SelectPosition.below;
-    protected positionChanged() {
-        this.positionAttribute = this.position;
+    public position?: SelectPosition;
+    protected positionChanged(
+        prev: SelectPosition | undefined,
+        next: SelectPosition
```

Comment:
Just to double check `@observable` doesn't discard undefined values or anything right? The type of next should be `SelectPosition | undefined`?

Reply:
As far as I know the `*Changed` functions' `prev` argument should always be typed like `sometype | undefined` because even if it has a default value, on the first run the `prev` will be `undefined` and the `next` will be the default value you assigned, but someone from the team feel free to correct me.

Reply:
I think @radium-v fixed up the missing select stuff and is going to push a fix for this. @simonxabris we'll work to get this merged today if possible! Thanks for this PR!